languages/elixir/exercises/concept/processes/.meta/example.ex

```diff
@@ -0,0 +1,24 @@
+defmodule TakeANumber do
```

Comment:
I feel that we're now getting into the more unique features of Elixir. Great to have exercises like this!